a10_neutron_lbaas/a10_context.py

```diff
@@ -91,7 +93,9 @@ class A10WriteContext(A10Context):
     def __exit__(self, exc_type, exc_value, traceback):
         if exc_type is None:
             try:
-                self.client.system.action.write_memory()
+                shared_partition = self.device_cfg.get("shared_partition", "shared")
+                self.client.system.action.write_active([self.partition_name, shared_partition])
```

Comment:
It's possible that `self.partition_name` is `None`. What happens in acos client if it tries to activate and write partition `None`?

This could maybe be simplified to

```
self.partition_names = []
...
self.partition_names = [name]
...
self.client.system.action.write_active(self.partition_names + [shared_partition])
```

Should we _always_ be writing a `self.device_cfg.get("shared_partition", "shared")` partition? What if they aren't using any partitions at all?

What will the acos `write_active` do if there are no partitions to activate and write?